python-sdk/src/astro/files/base.py

```diff
@@ -92,6 +93,22 @@ def create_from_dataframe(self, df: pd.DataFrame, store_as_dataframe: bool = Tru
         with smart_open.open(self.path, mode="wb", transport_params=self.location.transport_params) as stream:
             self.type.create_from_dataframe(stream=stream, df=df)
 
+    @property
+    def openlineage_dataset_namespace(self) -> Any:
+        """
+        Returns the open lineage dataset namespace as per
+        https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
+        """
+        return self.location.openlineage_dataset_namespace
+
+    @property
+    def openlineage_dataset_name(self) -> Any:
```

Comment:
Why is this returning `Any`?

Reply:
changed it to `str`. [Done](https://github.com/astronomer/astro-sdk/pull/1025/commits/7dad9a6fc4f0bcc5205f25fe51fb0b3f629a621e)